dashboard/app/models/course_offering.rb

```diff
@@ -35,6 +35,10 @@ class CourseOffering < ApplicationRecord
     with: KEY_RE,
     message: "must contain only lowercase alphabetic characters, numbers, and dashes; got \"%{value}\"."
 
+  ELEMENTARY_SCHOOL_GRADES = %w[K 1 2 3 4 5].freeze
```

Comment:
@tess323 I made up some grade bands here but let me know if I should change them! They'll also be easy to update in the future so no rush